docs/reference/buildx_history_ls.md

```diff
@@ -17,3 +17,48 @@ List build records
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+List completed builds recorded by the active builder. Each entry includes the
+build ID, name (if available), status, timestamp, and duration.
+
+By default, only records for the current builder are shown. You can filter
+results using flags.
+
+## Examples
+
+### List all build records for the current builder
+
+```console
+docker buildx history ls
+```
+
+### List only failed builds
+
+```console
+docker buildx history ls --filter status=error
```

Comment:
Can we show list all the available filters.

"ref", "repository", "status" - these allow `=` and `!=`
"startedAt", "completedAt", "duration" - allow `< > time`, eg. `duration>30s`.